### DISCUSSION TITLE
Deep Dive: Astroid Boilerplate Site 207

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#873] [Video Tutorials]
New Video is here with Article to follow early next week:

Next-Level Joomla and Astroid Article Integration
https://www.youtube.com/watch?v=gDktwy2ibws

Please like, share, comment etc.
And help get the word out that Astroid Rocks.
Thanks and Enjoy!

**Comment by paurray** [comment]
Article is here:
https://astroidframe.work/blog/astroid-hands-on/deep-dive-astroid-boilerplate-site-207

**Reply by sonvnn** [reply]
Awesome! Thanks @paurray. It would be great if the next topic would be Article Layout Builder :D

**Reply by paurray** [reply]
@Sonvnn
I have a tutorial about the Mega Menu in the works and I hope to get it out the door within the next week.

Early next year when the xmas dust settles I will start a 3xx series about the Article Layout Builder.

Kind of thinking of:

taking 2 example Article Layouts Apart:

1a)
https://demo.astroidframe.work/article-layouts/tours-example
1b)
https://demo.astroidframe.work/article-layouts/photography-example
2)
building an Article Layout from scratch
3)
and maybe importing/exporting an article layout

How does this sound? Any suggestions?

**Reply by sonvnn** [reply]
@paurray Thank you for contributing to Astroid. Issue 1,2 are really good and you can do it right away. Issue 3 is not possible at the moment. I will update the import and export function in the future.

**Comment by Banuk1616** [comment]
@paurray 

I really like your tutorials, thank you very much for your work with them.
